cpu/kinetis/periph/pwm.c

```diff
@@ -29,6 +29,8 @@
 #include "assert.h"
 #include "periph/pwm.h"
 
+#ifdef FTM0
```

Comment:
Can you add some comments on what this means? It looks strange that before it worked without this `#ifdef` and now we need it...